ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/UsersAndGroups_Test.java

```diff
@@ -47,6 +47,8 @@
     private static final Ds3ClientHelpers HELPERS = Ds3ClientHelpers.wrap(client);
     private static final String BUCKET_NAME = "Users_And_Groups_Test";
     private static final String TEST_ENV_NAME = "UsersAndGroups_Test";
+    private static final String USER_ID = "spectra";
+//    private static final String USER_ID = "Administrator";
```

Comment:
Instead of commenting this out, can we just get rid of it?

Reply:
I can change this if it makes more sense. My thought was going back and forth between the two versions.

Reply:
I see.  We're putting this in the 3_5_3 branch, though, yes?  If so, these changes shouldn't get merged into a 4.x branch.  Am I thinking about this right?  There's also another place with Administrator commented out.